packages/commonwealth/server/routing/router.ts

```diff
@@ -370,6 +372,21 @@ function setupRouter(
     getRelatedCommunitiesHandler.bind(this, serverControllers),
   );
 
+  registerRoute(
+    router,
+    'get',
+    '/communityStakes/:community_id/:stake_id',
+    getCommunityStakeHandler.bind(this, models, serverControllers),
+  );
+
+  registerRoute(
+    router,
+    'put',
+    '/communityStakes/:community_id/:stake_id',
+    passport.authenticate('jwt', { session: false }),
+    putCommunityStakeHandler.bind(this, models, serverControllers),
+  );
```

Comment:
Should these routes be behind a feature flag? (We did this previously for gating, it can just be an env var)

Reply:
I think this is fine to go in as discussed here:
https://github.com/hicommonwealth/commonwealth/pull/6274#issuecomment-1898864736

It won't cause any issues on prod anyways, because it will fail if the user doesn't have their namespace deployed.